.github/workflows/test.yml

```diff
@@ -6,46 +6,89 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            platform: linux-amd64
+            php-version: "8.3"
+            run-tests: true
+          - os: ubuntu-latest
+            platform: linux-arm64
+            php-version: "8.3"
+            run-tests: false
```

Comment:
We won't run tests for `linux-arm64`. I suppose you implemented this because `ubuntu-latest` is x64, but actually [GitHub has `ubuntu-24.04-arm` ](https://docs.github.com/en/actions/reference/runners/github-hosted-runners#standard-github-hosted-runners-for-public-repositories) we can use.

I'd also use `ubuntu-24.04` for x64 so both versions are uniform. Then we can remove the `run-tests:` logic from this file (and also the cross compilation).